battle_test.go

```diff
@@ -1062,3 +1063,61 @@ var _ = Describe("Status Conditions", func() {
 		})
 	})
 })
+
+var _ = Describe("In-a-pinch Berries", func() {
+	a1 := Agent(new(dumbAgent))
+	a2 := Agent(new(dumbAgent))
+	var holder *Pokemon
+
+	setup := func(itemId int) *Battle {
+		p1 := NewOccupiedParty(&a1, 0, GeneratePokemon(
+			PkmnCombusken,
+			WithLevel(25),
+			WithMoves(
+				GetMove(MoveFlamethrower),
+				GetMove(MoveDoubleKick),
+			),
+		))
+		holder = GeneratePokemon(
+			PkmnGrotle,
+			WithLevel(25),
+			WithMoves(
+				GetMove(MoveRazorLeaf),
+				GetMove(MoveBite),
+			),
+			WithIVs([6]uint8{1, 1, 1, 20, 1, 1}),
+		)
+		i := GetItem(itemId)
+		holder.HeldItem = &i
+		holder.CurrentHP = holder.Stats[StatHP] / 2
+		p2 := NewOccupiedParty(&a2, 1, holder)
+		b := NewBattle()
+		b.SetSeed(1337)
```

Comment:
Please use `TestRNG` or one of the presets instead of a seed here.

Reply:
This should be a quick fix, but changing the rng actually causing the test to be _more_ brittle.